src/components/services/Olivetin.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <Generic :item="item">
+    <template #content>
+      <p class="title is-4">{{ item.name }}</p>
+      <p class="subtitle is-6">
+        <template v-if="versionstring">
```

Comment:
Could you add a check on `item.subtitle` to display the subtitle insted of the version if set ? (most service card integration works like that, I forgot to mention it on gitea)

Reply:
Sure thing. Done in 1864e1d8253bbfef94ae732defb48ef3ff833750 . Except gitea... I'll clean that up in a separate PR, I guess?